Tasks/AzureFileCopy/AzureFileCopy.ps1

```diff
@@ -344,7 +353,8 @@ catch
 
     Write-Verbose $_.Exception.ToString() -Verbose
     $error = $_.Exception.Message
-    throw (Get-LocalizedString -Key "Upload to container: '{0}' in storage account: '{1}' with blobprefix: '{2}' failed with error: '{3}'" -ArgumentList $containerName, $storageAccount, $blobPrefix, $error)
+    $errorMessage = (Get-LocalizedString -Key "Upload to container: '{0}' in storage account: '{1}' with blobprefix: '{2}' failed with error: '{3}'" -ArgumentList $containerName, $storageAccount, $blobPrefix, $error)
```

Comment:
Do the localization in the throw error function. Not before passing into the error function as it makes the contract of the error function cleaner so in future if any error message is passed to this function unlocalized, it can localize it.